src/containers/OrgDetails.tsx

```diff
@@ -0,0 +1,252 @@
+  companyName: '',
+  country: '',
+  city: '',
+  streetName: '',
+  vat: '',
+  invoiceRequired: false
+};
+
+export const OrgDetails = () => {
+  // const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const { groupCheckout } = useAppState();
+
+  const organizerSchema = Yup.object().shape({
+    firstname: Yup.string()
+      .trim()
+      .matches(/^[a-z ,.'-]+$/i, 'Only a-z, A-Z chars')
+      .required('First name is required'),
+    lastname: Yup.string()
+      .trim()
+      .matches(/^[a-z ,.'-]+$/i, 'Only a-z, A-Z chars')
+      .required('Last name is required'),
+    phone: Yup.string()
+      .trim()
+      .matches(/^\+[1-9]\d{1,14}$/, 'Incorrect phone number')
+      .required('Phone number is required'),
+    email: Yup.string()
+      .trim()
+      .required('Email is required')
+      .matches(regexp.email, 'Incorrect email')
+      .email(),
+    companyName: Yup.string().trim(),
+    country: Yup.string()
+      .trim()
+      .test(
+        'is-allowed-country',
+        'Unknown country name',
+        (value) =>
+          value !== undefined && (value === '' || countriesOptions.includes(value))
+      ),
+    city: Yup.string().trim(),
+    streetName: Yup.string().trim(),
+    vat: Yup.string().trim()
+  });
+
+  const [vatValid, setVatValid] = useState<boolean>(false);
+  const [privacy, setPrivacy] = useState<boolean>(false);
+  const [error, setError] = useState<undefined | string>();
+
+  const methods = useForm<OrganizerInfo>({
+    resolver: yupResolver(organizerSchema),
+    defaultValues: groupCheckout?.organizerInfo ?? defaultValues
+  });
+
+  const {
+    watch,
+    handleSubmit,
+    formState: { isSubmitting }
+  } = methods;
+
+  useEffect(() => {
+    const subscription = watch((values) => {
+      dispatch({
+        type: 'SET_GROUP_CHECKOUT',
+        payload: {
+          ...groupCheckout,
+          organizerInfo: {
+            ...groupCheckout?.organizerInfo,
+            ...values
+          }
+        } as GroupCheckOut
+      });
+
+      if (values['vat'] && values['vat'].length === 11) {
+        const vatParsed = eeRegExp.exec(values['vat']) as EeRegExp | null;
+        if (vatParsed?.groups.NUM) {
+          isVatValid(vatParsed.groups.NUM)
+            .then(setVatValid)
+            // eslint-disable-next-line no-console
+            .catch(console.error);
+        }
+      }
+    });
+    return () => subscription.unsubscribe();
+  }, [watch]);
+
+  const onSubmit = useCallback(
+    async (values: OrganizerInfo) => {
+      logger.info('Submit organizer data', values);
+      try {
+        setError(undefined);
+
+        // navigate('/checkout/' + ...);
+      } catch (error) {
+        const message = (error as Error).message || 'Unknown error';
+        setError(message);
+      }
+    },
+    [groupCheckout]
+  );
+
+  return (
+    <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
+      {error && <Alert severity="error">{error}</Alert>}
```

Comment:
If this is according to design, it should be ofc kept like this, but I am used that error messages are shown under the button with forms.

Reply:
this code has been copied from the UserInfo page. I guess we need to follow the same logic and same UX patterns in the app. If this behaviour is not OK, we have to change it everywhere.

Reply:
the design wireframes do not contain any suggestions about where errors should be displayed. If we will change it here, also we need to change it in the UserInfo form